ext/mkrf_conf.rb

```diff
@@ -0,0 +1,24 @@
+require 'rubygems'
+require 'rubygems/command.rb'
+require 'rubygems/dependency_installer.rb' 
+begin
+  Gem::Command.build_args = ARGV
+  rescue NoMethodError
+end 
+inst = Gem::DependencyInstaller.new
+
+
+begin
+  if RUBY_VERSION < "1.9.3"
+    inst.install "activerecord", ">2.0", "< 4.0"
+    inst.install 'i18n', "< 0.7"
+  else
+    inst.install "activerecord", "> 4.0"
```

Comment:
This is the same logic as it was in gemspec. Ok, not exacly the same as you check if the Ruby version is not greather that 1.9.3

```
RUBY_VERSION =~ /^1\.8|^1.9\.[1|2]/
```
I thought it could be first step: to move logic from gmespec (where it doesn't work as expected) to extension, where it should work.

Unfortunately, it doesn't work for me :( The activerecord is not installed as dependency in any version. I have no idea why, I'm trying to figure out what's going on. No luck so far.